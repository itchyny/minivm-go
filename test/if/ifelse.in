if true
  a = 10
  print a
else
  b = 11
  print b
end

c = 12
print c

if false
  d = 13
  print d
else
  if false
    print 100
  else
    e = 14
    print c * e
  end
end
