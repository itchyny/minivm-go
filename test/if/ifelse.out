10
12
168
